pilot/pkg/bootstrap/server.go

```diff
@@ -206,18 +243,50 @@ func (s *Server) Start(stop <-chan struct{}) error {
 		}
 	}
 
+	// grpcServer is shared by Galley, CA, XDS - must Serve at the end, but before 'wait'
+	go func() {
+		if err := s.grpcServer.Serve(s.grpcListener); err != nil {
```

Comment:
Start before cache sync may cause problem see https://github.com/istio/istio/issues/12264

Reply:
There is a chicken-and-egg if galley is used ( cache sync will use the grpc server ). 
And readiness server should not mark the pod as active until cache sync is done.

Reply:
But we donot have readiness probe for pilot or istiod yet, do we?

Reply:
We do have a readiness probe for pilot

Reply:
I see that.